manage.py

```diff
@@ -10,6 +10,7 @@
 import sqlalchemy as sa
 from flask_script import Server
 from flask_script import Manager
+import requests
```

Comment:
Thank you!  There's another import missing as well, the builtin `re` module (see around line 421).